setup.py

```diff
@@ -17,14 +17,22 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
+import os
 
 from setuptools import setup, find_packages
 
 from unstructured.__version__ import __version__
 
+cwd = os.path.dirname(os.path.abspath(__file__))
+
+# Read in README.md for our long_description
+with open(os.path.join(cwd, "README.md"), encoding="utf-8") as f:
+    long_description = f.read()
+
 setup(
     name="unstructured",
     description="A library that prepares raw documents for downstream ML tasks.",
+    long_description=long_description,
```

Comment:
We should be able to do this without importing `os`, as seen in `transformers` [here](https://github.com/huggingface/transformers/blob/main/setup.py#L416-L417). While we're in here, let's also add `keywords`, `url`, `python_requires`, and `classifiers`.  For `classifiers` we can borrow what the classifiers from `transformers`, except drop `Programming Language :: Python :: 3.7` and change development status to `Development Status :: 4 - Beta`

Reply:
@MthwRobinson  What keywords you suggest? I think maybe: 
"NLP PDF HTML parsing preprocessing"

Reply:
I'd add `CV` and `XML`, the others look good to me!